test/elm/clinical/data.cql

```diff
@@ -44,6 +50,10 @@ define InListOfCodesExpressionRef: ListOfCodes in "Female"
 define InWrongListOfCodes: WrongListOfCodes in "Female"
```

Comment:
Quite a few of these tests are actually for `AnyInValueSet` and should be moved up there while we are here.